src/box/sql/parse_def.h

```diff
@@ -235,6 +235,16 @@ struct sql_parse_column_list {
 	uint32_t n;
 };
 
+/** Description of the table being created. */
+struct sql_parse_table {
+	/** Table name. */
+	struct Token name;
+	/** Table engine name. */
+	struct Token engine_name;
+	/** IF NOT EXISTS flag. */
+	bool if_not_exists;
+};
```

Comment:
Further on the point about struct-per-statement - I would expect this struct to have column list, index list, fk list, etc. Not store that in the generic `struct Parse`. I know it wasn't like that even before, but it would be very good to do so.